src/Build/Evaluation/LazyItemEvaluator.LazyItemOperation.cs

```diff
@@ -110,14 +110,14 @@ protected void DecorateItemsWithMetadata(ICollection<I> items, ImmutableList<Par
 
                     // Do not expand properties as they have been already expanded by the lazy evaluator upon item operation construction.
                     // Prior to lazy evaluation ExpanderOptions.ExpandAll was used.
-                    const ExpanderOptions metadataExpansionOptions = ExpanderOptions.ExpandMetadata | ExpanderOptions.ExpandItems;
```

Comment:
Oh, you're right, I had not seen / realized that the metadata was inside a property function call.
My bad, the fix is good. Good bye optimization :)